File: Additional Sheet (Condition)/F.cpp
#include <iostream>

using namespace std;
int main()
{
    int a,b;
   
    cin>>a>>b;
   

     if((a>b&&b!=1)||(a!=b&&a==1)){
         cout<<"Alice"<<endl;
    }
   
    else if((b>a&&a!=1)||(a!=b&&b==1))  {
        cout<<"Bob"<<endl;
    }
    else{
        cout<<"Draw"<<endl;
    }
    
  
    
    return 0;
}

File: Additional Sheet (Condition)/F.py
x,y = list(map(int,input().split()))
if x == y:
    print('Draw')
elif x == 1:
    print('Alice')
elif y == 1:
    print('Bob')
elif x > y and y != 1:
    print('Alice')
elif x < y and x != 1:
    print('Bob')
